src/components/Comment.jsx

```diff
@@ -0,0 +1,13 @@
+export const Comment = ({ comment }) => {
+  const date = new Date(comment.created_at);
+  return (
+    <div className="comment">
+      <p>Author: {comment.author}</p>
+      <p>{comment.body}</p>
+      <p>Posted on: {date.toDateString()}</p>
+      <p>Votes: {comment.votes}</p>
+      {/* <p>Comment ID: {comment.comment_id}</p>
```

Comment:
Yeah no need to have comment_id there, a user doesn't need it. I would just delete this